api-remove.go

```diff
@@ -249,7 +271,7 @@ func (c Client) removeObjects(ctx context.Context, bucketName string, objectsCh
 		}
 		if err != nil {
 			for _, b := range batch {
-				errorCh <- RemoveObjectError{ObjectName: b, Err: err}
+				errorCh <- RemoveObjectError{ObjectName: b.Key, Err: err}
```

Comment:
shouldn't the version attempted to be deleted be sent here in the errorCh too?

Reply:
> shouldn't the version attempted to be deleted be sent here in the errorCh too?

Indeed, good catch!